x-pack/plugins/security_solution/public/management/components/console/components/history_item.tsx

```diff
@@ -7,22 +7,26 @@
 
 import React, { memo, PropsWithChildren } from 'react';
 import { EuiFlexItem } from '@elastic/eui';
+import styled from 'styled-components';
 import { useTestIdGenerator } from '../../../hooks/use_test_id_generator';
 import { useDataTestSubj } from '../hooks/state_selectors/use_data_test_subj';
 
+const StyledEuiFlexItemHistoryItem = styled(EuiFlexItem)`
+  flex-basis: 99% !important;
```

Comment:
what does the 99% do?

Reply:
For whatever reason, the item was causing a horizontal scroll bar to appear in some cases. I'll probably come back around to try to figure out what the root cause is. Maybe its the fact that I don't have `overflow: hidden` on the element. Will try that

Reply:
Ohhh Good one. I will try that and see if perhaps that addresses the `99%` setting here as well.